### PR TITLE
Update tenant-rename.md

### PR DESCRIPTION
**Pull request by WayneEwington**
Clarified that this feature is only available for customers that have been accepted into the Private Preview.
Called out that third-party applications including backup solutions should be confirmed to support tenant renames.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/tenant-rename.md [hunk 1]
```diff
@@ -34,7 +34,10 @@ description: "Learn how to change your SharePoint domain name using PowerShell"
 It's now possible to change the SharePoint domain name for your organization in Microsoft 365. For example, if the name of your organization changes from "Contoso" to "Fabrikam," you can change `contoso.sharepoint.com` to `fabrikam.sharepoint.com`.
 
 > [!IMPORTANT]
-> This preview is rolling out to organizations with the fewest sites first. It might not be available yet for your organization. This feature isn't available for organizations that have set up Multi-geo.
+> This feature is only available for customers that have been accepted into the Private Preview program. If you would like to nominate yourself into the Private Preview you can do so at https://aka.ms/SPOTenantRenameNomination. Please note that the nomination does not necessarily mean that you will be accepted. We are reviewing these on a case-by-case basis and will only be in contact if you are accepted.
+
+> [!IMPORTANT]
+> This feature is available isn't available for organizations that have set up Multi-geo.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:40]:
```suggestion
> This feature is not available for organizations that have set up a multi-geo.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @WayneEwington, kindly commit mu suggested changes if you agree with them. Many thanks.

File: SharePoint/SharePointOnline/tenant-rename.md (hunk 1 above)
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/tenant-rename.md:40]:
```suggestion
> - This feature is not available for organizations that have set up a multi-geo.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**